### PR TITLE
Create 01-introduction.Rmd

### PR DESCRIPTION
**Pull request by jannes-m**
Here, I just delete a few typos, and make a few recommendations/remarks (qualitative GIS, interfaces).

### CODE REVIEW

File: 01-introduction.Rmd [hunk 1]
```diff
@@ -5,7 +5,7 @@ Suggested titles were *Geography with R* and *R for GIS*, each of which has adva
 The former conveys the message that it's not just about spatial data: 
 non-spatial attribute data are inevitably interwoven with geometry data, and Geography is about more than where something is on the map.
 The latter option communicates that this is a book about using R as a *Geographic Information System* (GIS), to perform spatial operations on *geographic data* [@bivand_applied_2013].
-However, the term GIS has some negative connotations (see Table \@ref(tab:gdsl)) and fails to communicate one of R's greatest strengths:
+However, the term GIS has some negative connotations <!-- I see what you mean by negative connotations but I had to think immediately of qualitative GIS and critical GIS, i.e., scientists from the humanities criticising GIS technologies. Think of the so-called "GIS crisis", positivism, make-believe objectivity, etc.--> (see Table \@ref(tab:gdsl)) and fails to communicate one of R's greatest strengths:
```
Comment (Robinlovelace) [inline 01-introduction.Rmd]: Thanks for this comment. @Nowosad maybe it's better to clarify what we mean here. Note the great Stan Openshaw was talking about a crisis in GIS back in 1991. Worth citing this: http://journals.sagepub.com/doi/abs/10.1068/a230621?journalCode=epna to be clear what we mean?

File: 01-introduction.Rmd [hunk 2]
```diff
@@ -22,24 +22,23 @@ The title was chosen because this book is about more than routine spatial data p
...
 young field^[Geocomputation
 has a ~30 year history dating back to the first [conference](http://www.geocomputation.org/) on the subject in 1996 (at the University of Leeds where the concept for this book was dreamt up) which was followed by a flurry of subsequent
 publications.]
 but methods published under the geocomputational banner have influenced the direction of geographical research, as we will see in subsequent chapters.
 Algorithms are powerful tools that can become highly complex.
 However, all algorithms are composed of smaller parts.
-By teaching these foundations we aim to empower you.
-Creating your own solutions to geographic data problems can feel like breaking free from the metaphorical 'glass ceiling' imposed by GUI-based proprietary geographic information systems (see Table \@ref(tab:gdsl) for a definition of GUI).
+By teaching these foundations we aim to empower you to create your own solutions to geographic data problems. This can feel like breaking free from the metaphorical 'glass ceiling' imposed by GUI-based proprietary geographic information systems (see Table \@ref(tab:gdsl) for a definition of GUI).
```
Comment (Robinlovelace) [inline 01-introduction.Rmd]: This is a clear improvement.

File: 01-introduction.Rmd [hunk 3]
```diff
@@ -51,7 +50,7 @@ knitr::kable(x = d, caption = "Differences in emphasis between the fields of Geo
 In this book we treat R as a 'tool for the trade'.
 Early geographers used a variety of tools including rulers, compasses and sextants to advance knowledge about the world. 
 <!--nowadays part-->
-It is important to remember that while R is a powerful tool, especially when interfaced with other software such as [GDAL](http://www.gdal.org/) and [PostGIS](http://postgis.net/), other tools may be better for certain tasks.
+It is important to remember that while R is a powerful tool, especially when interfaced with other software such as [GDAL](http://www.gdal.org/) and [PostGIS](http://postgis.net/), other tools may be better for certain tasks. <!-- What exactly do you mean? Which tasks? Is there a geocomputational task for which we do not have an interface? So maybe rephrase along the lines that R is not a GIS and therefore not suited for some geoprocessing tasks but that interfaces close this gap. -->
```
Comment (Robinlovelace) [inline 01-introduction.Rmd]: Yes I think it's worth rephrasing along those lines.

**Comment by jannes-m** [comment]
Maybe one last remark regarding the introduction or preface (I don't know what you are planning). Maybe it would be useful to cite aside from the ASDAR book also [Chris Brundson's book](https://uk.sagepub.com/en-gb/eur/an-introduction-to-r-for-spatial-analysis-and-mapping/book241031) to better emphasize why a new book is needed. Then you could show what your book covers (simple features, raster, stars? (https://github.com/r-spatial/stars), R/GIS integration) and what not (point pattern analysis, geostatistical analyses (kriging), spatial regression models, spatial cross-validation, etc.). Other books which might be of interest in this context:

- https://www.packtpub.com/big-data-and-business-intelligence/learning-r-geospatial-analysis ( I have to admit I haven't read this one)
- http://oscarperpinan.github.io/spacetime-vis/

Maybe a bit off topic but interesting for showing what the book not covers:
- of course your book on geospatial simulations
- https://www.crcpress.com/Spatial-Point-Patterns-Methodology-and-Applications-with-R/Baddeley-Rubak-Turner/p/book/9781482210200
- http://spatial-analyst.net/book/
- http://www.springer.com/us/book/9780387329079
- http://eu.wiley.com/WileyCDA/WileyTitle/productCd-1118326555.html
- http://www.highstat.com/BGSpattemp.htm

**Comment by Robinlovelace** [comment]
Many thanks for this, very useful. Will get on it when I'm Nijmegen. Cycling there from Utrecht today.

**Comment by jannes-m** [comment]
Hey there, 
good idea! I will get on my bike now also, and maybe before I will head into the bike workshop. It's been a while but a few years ago, I was working on a bike package. If you are into bikes and bike building have a look at https://github.com/jannes-m/velo, might be useful.
